README.md

```diff
@@ -30,6 +30,32 @@ $ ros2 bag info -s mcap path/to/your_recording.mcap
 ```
 
 
+### Writer Configuration
+
+To configure details of the MCAP writer for `ros2 bag record`, use the `--storage-config-file` options to provide a YAML file describing `mcap::McapWriterOptions`. See the following example providing some options.
+
+See [McapWriterOptions declaration](https://github.com/foxglove/mcap/blob/main/cpp/mcap/include/mcap/writer.hpp#L18) for all fields. Any omitted values will be left as default value.
```

Comment:
I feel like this should link to the yaml name -> option mapping in `mcap_storage.cpp`, rather than to the MCAP repo directly. This avoids issues where we add a new storage option to the MCAP library in `main` without plumbing it through to the YAML definition for a particular plugin.version.

Reply:
Fair point - though the comments in the writeroptions are necessary for semantics/context. But I can add a link to each to explain what's mapped through.

Ideally of course this stays up to date. Maybe it would make sense for this to link to the specific release version of the mcap source that this plugin is pulling in? That would require manual update though, I don't think there's a way to make the markdown smart about it.